app/grandchallenge/algorithms/templates/algorithms/algorithm_form.html

```diff
@@ -25,3 +25,17 @@ <h2>{{ object|yesno:"Update,Create" }} Algorithm</h2>
     {% crispy form %}
 
 {% endblock %}
+
+{% block script %}
+    {{ block.super }}
+    <script>
+        if (document.getElementById("id_display_editors").value == ''){
+            document.getElementById("id_display_editors").classList.add('is-invalid');
+            const alert = document.createElement('span');
+            alert.id = "error_1_id_display_editors";
+            alert.className = "invalid-feedback"
+            alert.innerHTML = "<strong>This field is required.</strong>";
+            document.getElementById("div_id_display_editors").children[1].insertBefore(alert, document.getElementById("hint_id_display_editors"));
+        }
+    </script>
```

Comment:
Unlike the other required fields, the `display_editors` field was not marked red after validation (only the alert was shown). This is a work around to have the same markup as for the other required fields. This also makes finding the missing input easier for the user.